officers/two_of_ten.json

```diff
@@ -37,9 +38,14 @@
       "requirements": {
         "badges": 0,
         "credits": 0,
-        "others": [],
+        "others": [
+          {
+            "name": "Active Nanoprobe",
+            "value": 3000
+          }
+        ],
         "shards": 65,
-        "xp": 3000
+        "xp": 0
```

Comment:
What happened here? XP seems to be reduced to 0 for 2of10?

Reply:
From how I read the data the unimatrix twelve officers just have `Active Nanoprobe` instead of `XP`. I previously didn't see that so I had that column in XP which I now reset to 0 and added the other resource

Reply:
OK cool, thanks